week3/60-kvadratnoie-uravnieniie-2/task.py

```diff
@@ -0,0 +1,52 @@
+import math
+
+
+class rootCounts:
+    NaN = "3"
+    Two = "2"
+    One = "1"
+    Zero = "0"
+
+
+def if_n_is_integer_convert_to_int(n):
+    if n.is_integer():
+        return int(n)
+    else:
+        return n
+
+
+def calculate_roots(a, b, c):
+    D = b ** 2 - 4 * a * c
+    if D == 0:
+        x = if_n_is_integer_convert_to_int(-b / (2 * a))
+        return rootCounts.One, str(x)
```

Comment:
str(x) is part of printing results - not part of solving equation. try to return numbers from ewquation solver. try to format result for output independently

Reply:
done